span.go

```diff
@@ -146,7 +146,7 @@ type FinishOptions struct {
 	//
 	// If specified, the caller hands off ownership of BulkLogData at
 	// FinishWithOptions() invocation time.
-	BulkLogData []*LogData
+	BulkLogData []LogData
```

Comment:
Yes, I saw this... it's the only "interesting" (non-lint) change to the public API. I don't know whether it's reasonable to assume that the caller will always know how many `LogData`s they want to add at once. Nor do I know whether the above actually matters... It looks like @tschottdorf's benchmarks suggest the allocations are worse than the copying in the `RawSpan` case at least.